include/rmm/mr/device/detail/arena.hpp

```diff
@@ -33,24 +33,48 @@ namespace mr {
 namespace detail {
 namespace arena {
 
-constexpr std::size_t superblock_size = 1u << 26u;  ///< Size of a superblock (64 MiB).
+/// Minimum size of a superblock (256 KiB).
+constexpr std::size_t minimum_superblock_size = 1u << 18u;
```

Comment:
You have changed this 3 times, possibly more. Should it be an optional parameter of the class?

Reply:
It's used by both `block` and `arena`, so can't really be a non-type template parameter for either.